src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -37,9 +37,9 @@ class PexBinary(TemplatedExternalTool):
     name = "pex"
```

Comment:
This was not a clean pick due to the `options_scope` name change to "pex-cli" on main.